Project.toml

```diff
@@ -0,0 +1,28 @@
+name = "AWSS3"
+uuid = "1c724243-ef5b-51ab-93f4-b0a88ac62a95"
+authors = ["Sam O'Connor"]
+version = "0.6.0"
+
+[deps]
+AWSCore = "4f1ea46c-232b-54a6-9b17-cc2d0f3e6598"
+Base64 = "2a0f44e3-6c83-55bd-87e4-b1978d98bd5f"
+DataStructures = "864edb3b-99cc-5e75-8d2d-829cb0a9cfe8"
+Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
+EzXML = "8f5d6c58-4d21-5cfd-889c-e3ad7ee6a615"
+FilePathsBase = "48062228-2e41-5def-b9a4-89aafe57970f"
+HTTP = "cd3eb016-35fb-5094-929b-558a96fad6f3"
+MbedTLS = "739be429-bea8-5141-9913-cc70e7f3736d"
+Retry = "20febd7b-183b-5ae2-ac4a-720e7ce64774"
+SymDict = "2da68c74-98d7-5633-99d6-8493888d7b1e"
+UUIDs = "cf7118a7-6976-5b1a-9a39-7adc72f591a4"
+XMLDict = "228000da-037f-5747-90a9-8195ccbf91a5"
+
+[compat]
+AWSCore = ">= 0.5"
+julia = "1"
```

Comment:
Need compat version for FilePathsBase

Reply:
That'll come when I tag a new release

Reply:
I don't understand

Reply:
The compat will be the next version of FilePathsBase which I haven't released yet.